core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -264,6 +264,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
   val tachyonFolderName = externalBlockStoreFolderName
 
   def isLocal: Boolean = (master == "local" || master.startsWith("local["))
+  def isStopped: Boolean = stopped.get()
```

Comment:
It seems to me it should be fine to be public, since we can always have isStopped